src/locale.cpp

```diff
@@ -664,10 +665,12 @@ locale::id::__get()
     return __id_ - 1;
 }
 
+#include <builtins.hpp>
```

Comment:
Do you really want the #include this far down in the file?  I know it is legal, but it makes it hard to see what include files are used with this file.  If there is a reason for the include to be this far down then Ok.

Reply:
There was no real reason. I just put it there when I was debugging the code. Moved up.